### PR TITLE
Fixes bug #7 which allowed for multiple simultaneous pane monitoring

### PR DESCRIPTION
**Pull request by rickstaa**
This pull request fixes bug #7 which allowed the same pane to be monitored multiple times. Additionally with the new fix user are now also able to monitor multiple panes in different windows and sessions at the same time.

### CODE REVIEW

File: scripts/cancel.sh [hunk 1]
```diff
@@ -1,21 +1,34 @@
 #!/usr/bin/env bash
 ## -- Cancel monitoring script
+# Used to cancel the current  pane monitor job
 
-# get id of the current active pane
-PANEID=$(tmux list-panes | grep "active" | awk -F\] '{print $3}' | awk '{print $1}')
-PID_DIR=~/.tmux/notify
+# Get current directory
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```
Comment (ChanderG) [inline scripts/cancel.sh:6]: Any reason for not using just using `CURRENT_DIR=$(dirname $BASH_SOURCE)`?

**Review by rickstaa — COMMENTED** [review]
@ChanderG I quickly double-checked why the `CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && PWD )"` syntax is used in most bash scripts over the simpler `CURRENT_DIR=$(dirname $BASH_SOURCE)` syntax. The reason is explained in this [blog post](https://www.binaryphile.com/bash/2020/01/12/determining-the-location-of-your-script-in-bash.html). The longer version always get the absolute path and therefore also works when the working directory is changed before you use the path. As this doesn't seem to be the case in our scripts, we can also use the shorter form. Nevertheless, I think it does not hurt the use of the longer version to prevent future possible errors if we change the scripts in the future. Let me know what you prefer so that I can merge this pull request.

**Comment by rickstaa** [comment]
@ChanderG I fixed some last bugs and merged the changes into the master branch. Feel free to let me know if you want to use the simpler directory fetch syntax.

**Comment by ChanderG** [comment]
I wasn't aware of the use of the longer version. Makes sense.

**Comment by rickstaa** [comment]
> I wasn't aware of the use of the longer version. Makes sense.

I wasn't familiar with it either. There is also the `cd - p` version to make it work with symlinks.

**Pull request merged**